### DISCUSSION TITLE
Do inline projections support event metadata or not?

### DISCUSSION THREAD

**Discussion by thomaslevesque** [JasperFx/marten#3431] [Q&A] [answered]
The documentation seems to contradict itself.

[Here](https://martendb.io/events/projections/#inline-projections) it says:
> First off, be aware that event metadata (e.g. stream version and sequence number) are not available during the execution of inline projections. If you need to use event metadata in your projections, please use asynchronous or live projections.

But [here](https://martendb.io/events/projections/inline.html) it says:
> Note that the inline projection is able to use the [event metadata](https://martendb.io/events/metadata.html) at the time the inline projection is executed. That was previously a limitation of Marten that was fixed in Marten V4.

So which is it? I suspect the first one is incorrect, and just wasn't updated when this limitation was removed?

**Comment by mysticmind** [comment, marked as answer]
We have to update the docs to remove the following:

> First off, be aware that event metadata (e.g. stream version and sequence number) are not available during the execution of inline projections. If you need to use event metadata in your projections, please use asynchronous or live projections.

If you want to give it a shot to send a PR with the correction to docs. Please do.

**Reply by thomaslevesque** [reply]
Thanks! I'll try to send a PR, but no promise. I'm only experimenting with Marten at the moment 😉

**Reply by thomaslevesque** [reply]
https://github.com/JasperFx/marten/pull/3432
